samples/containers/redis/README.md

```diff
@@ -1,6 +1,9 @@
 Running Redis with SGX-LKL-OE
 =============================
 
+0. Make sure that you have installed ``redis-cli``. In Ubuntu, the package that
```

Comment:
I think the list should start at 1.

This "0" is weird to me.

Reply:
Done.